### ISSUE TITLE
Docs for V6 to V7 Server SDK are not helpful

### ISSUE THREAD

**Issue by thejeff77** [launchdarkly/ld-relay#273]
Hey there,

I'm reading this page: https://docs.launchdarkly.com/sdk/server-side/java/migration-6-to-7

Trying to figure out how to migrate LDUser to something else. 

The only link in there that seemed like it might help me figure out how to migrate is the "Understanding what was removed" which goes nowhere.

Furthermore, this migration guide says little more than upgrade to 6 before going to 7 and that LDUser was removed with no helpful information on how you're actually supposed to _**migrate**_ which is supposedly the purpose of the page.

Thanks for updating this page with any useful migration information, and adding links that go places.

Jeff

**Is this a support request?**
This issue tracker is maintained by LaunchDarkly SDK developers and is intended for feedback on the SDK code. If you're not sure whether the problem you are having is specifically related to the SDK, or to the LaunchDarkly service overall, it may be more appropriate to contact the LaunchDarkly support team; they can help to investigate the problem and will consult the SDK team if necessary. You can submit a support request by going [here](https://support.launchdarkly.com/hc/en-us/requests/new) or by emailing support@launchdarkly.com.

Note that issues filed on this issue tracker are publicly accessible. Do not provide any private account information on your issues. If your problem is specific to your account, you should submit a support request as described above.

**Describe the bug**

**To reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Logs**
If applicable, add any log output related to your problem.

**SDK version**
The version of this SDK that you are using.

**Language version, developer tools**
For instance, Go 1.11 or Ruby 2.5.3. If you are using a language that requires a separate compiler, such as C, please include the name and version of the compiler too.

**OS/platform**
For instance, Ubuntu 16.04, Windows 10, or Android 4.0.3. If your code is running in a browser, please also include the browser type and version.

**Additional context**

**Comment by cwaldren-ld** [comment]
Hi @thejeff77, did you close this because you found the information you need? Want to make sure you aren't still hunting.

All the links on that page do go places and we take seriously any lack of documentation. 

If it was a misunderstanding, happy to take that feedback and work with our documentation team to improve.

Edit: I see the anchor link you are talking about. On desktop it probably appears to not work, but if you were on mobile (or had a really tiny screen width) it'd jump the page.

**Comment by thejeff77** [comment]
Yeah I noticed that the link went to a heading at the bottom of the page.. which is sort of weird to do with such a short page where that other section won't go off the page.

Would have been nice to have a one-liner on here with:

Replace `LDUser(mykey)` with `LDContext.create(mykey)`

**Comment by mmrj** [comment]
Hi @thejeff77 , thanks for the feedback! We've updated all migration guides for SDK versions that remove `LDUser` to include a quick code sample for creating contexts and, for server-side SDKs, evaluating flags for them. Great suggestion.

Having the link in the intro paragraph jump do a heading down the page is our standard format for migration guides...and hopefully makes more sense today than yesterday now that we've released [migration flags](https://docs.launchdarkly.com/home/flag-types/migration-flags) to everyone as part of our [Galaxy event](https://launchdarkly.com/galaxy/). The relevant content was previously flagged off in the [Java SDK v6 to v7 migration guide](https://docs.launchdarkly.com/sdk/server-side/java/migration-6-to-7) but is now present, so there's actually something to jump over : )
